src/Evolution/Executables/GeneralizedHarmonic/EvolveGhBinaryBlackHole.hpp

```diff
@@ -180,6 +180,28 @@ namespace Parallel {
 template <typename Metavariables>
 class CProxy_GlobalCache;
 }  // namespace Parallel
+
+// We don't use the Tags::TimeAndPreviousCompute in
```

Comment:
Will this also be an issue for BNS when a BH forms? If so, should we handle this either in `TimeAndPrevious` or move this to `Time/Tags/`?

Reply:
No this won't be an issue for BNS because there will only be one horizon finder there.

Reply:
Okay, sounds good!